docs/quickstart.rst

```diff
@@ -9,7 +9,7 @@ You can try funcX on `Binder <https://mybinder.org/v2/gh/funcx-faas/examples/HEA
 Installation
 ------------
 
-**funcX** comes with two components: the **endpoint**, a user-managed software agent that must be deployed on a compute resource to make it accessible for function execution; and the **funcX client**, which provides a Python API for registration, execution, and management of functions across **endpoints**.
+**funcX** comes with two components: the **endpoint**, a user-managed software agent that must be deployed on a compute resource to make it accessible for function execution, and the **funcX client**, which provides a Python API for registration, execution, and management of functions across **endpoints**.
```

Comment:
The original punctuation reads better because the individual list items have commas in them (and using a semicolon in this case is grammatically correct, see [here](https://www.merriam-webster.com/words-at-play/a-guide-to-using-semicolons))

Reply:
Yes I think you are right.  it is a separation of the items in a list after a colon.